packages/react-router/tests/link.test.tsx

```diff
@@ -3030,6 +3030,47 @@ describe('Link', () => {
 
     expect(ErrorComponent).not.toHaveBeenCalled()
   })
+
+  test.only('when linking to self with from prop set and param containing a slash', async () => {
```

Comment:
please remove the `.only`

Reply:
Oh, definitely didn't mean to include that in the PR. Good catch!